### PR TITLE
chore(setup): Setup UX service lint

### PR DESCRIPTION
**Pull request by jmorgan74**
N.B. I'm aware that we can configure one eslint, pretter ruleset for both api and ux services at the top level but I've never done it, still learning. :-)

- Add .eslintrc, .prettierrc and tsconfig files to UX sub dir.

### CODE REVIEW

File: ux-microservice/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (MrSimonEmms) [inline ux-microservice/.eslintrc:1]: This is the Typescript config - let me work out a JS version of this and I'll send it you

File: ux-microservice/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (MrSimonEmms) [inline ux-microservice/tsconfig.json:1]: If using JS-not-TS, then we don't need this file

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]

File: ux-microservice/tsconfig.json (hunk 2 above)
Comment (jmorgan74) [reply ux-microservice/tsconfig.json:1]: yes but we don't know yet until tomorrow

Comment (jmorgan74) [reply ux-microservice/tsconfig.json:1]: I was hoping to go TypeScript with this too so just got it ready.

File: ux-microservice/.eslintrc (hunk 1 above)
Comment (jmorgan74) [reply ux-microservice/.eslintrc:1]: am aware, see comment below.

**Comment by MrSimonEmms** [comment]
@jmorgan74 I think this should work for you. I need to work on the trailing comma in a function issue, but that's just a rule with I need to disable

```
{
  "parserOptions": {
    "ecmaVersion": 6,
    "sourceType": "module"
  },
  "extends": [ "airbnb-base", "prettier" ],
  "plugins": [ "jest", "prettier" ],
  "root": true,
  "ignorePatterns": [ "node_modules/**", "dist/**" ],
  "rules": {
    "prettier/prettier": "error"
  },
  "env": {
    "es2021": true,
    "node": true,
    "jest/globals": true
  }
}
```

**Review by MrSimonEmms — APPROVED** [review]

**Pull request closed without merging**